src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -321,17 +321,24 @@ def test_metadata_change_triggers_content_review(self):
         data = self.get_dict()
         addon = self.addon = self.get_addon()
         AddonApprovalsCounter.approve_content_for_addon(addon=addon)
-        assert AddonApprovalsCounter.objects.get(
+        old_content_review = AddonApprovalsCounter.objects.get(
             addon=addon).last_content_review
+        assert old_content_review
 
         # make the edit
         response = self.client.post(self.describe_edit_url, data)
         assert response.status_code == 200
         addon = self.addon = self.get_addon()
 
-        # last_content_review should have been reset and the metadata updated
-        assert not AddonApprovalsCounter.objects.get(
-            addon=addon).last_content_review
+        if self.listed:
```

Comment:
Does this test definitely run with an addon with only unlisted versions at some point?

Reply:
Yes, via the child class `TestEditDescribeUnlisted`. Without that if/else block, the test fails.